R/calc_views.R

```diff
@@ -88,7 +91,7 @@ which_reals <- function(values, na.value){
   
 }
 set_view_window <- function(views, param, na.value=NA, remove=TRUE, ignore=NULL){
-  view_i <- which(names(views) %in% "view")
+  view_i <- grep('view.', names(views))
```

Comment:
I was thinking more that it would stay as-is, but would happen in only one place: `view_indices <- function(gsplot)` clean up to come on that in the future?